src/Nethermind/Nethermind.Merge.Plugin/IMergeConfig.cs

```diff
@@ -60,7 +60,7 @@ public interface IMergeConfig : IConfig
             - A positive number to release memory after that many Engine API calls
 
 
-            """, DefaultValue = "75")]
+            """, DefaultValue = "25")]
```

Comment:
Why the need? Wouldn't it be bad for normal processing?

Reply:
Actually reduces GC time becuase is much smaller each time; rather than having a very large heap to examime.  Difference is peak usage of 4GB or 9GB on saw tooth allocated memory